module/Company/src/Company/Service/Company.php

```diff
@@ -850,7 +1115,7 @@ public function getAcl()
     }
 
     /**
-     * Get the default resource ID.
+     * Get the default resource Id.
```

Comment:
We should write a section in the Wiki on how we want to capitalize acronyms in comments/method names. I understand that everybody has a personal preference, but we risk undoing each other’s changes. Additionally, it pollutes the PR with unrelated code changes.

Reply:
@armandduijn I totally agree with you, normally I would not change this. But in this case, in the changes originally made, both capitalizations were used. To directly streamline it (in this module) I opted to change everything in one go.

As PSR-1 explicitly does not mention how to capitalize properties, I used the naming convention for (albeit) classes from the PHP. In this case `Id` is really used for `Identifier` whereas `ID` is used for `Identity Document`.

We should add a section to the Wiki to resolve this in the future.